crates/bootstrap_client/tests/connect.rs

```diff
@@ -0,0 +1,36 @@
+use kitsune2_bootstrap_srv::{BootstrapSrv, Config};
+use kitsune2_core::{Ed25519LocalAgent, Ed25519Verifier};
+use kitsune2_test_utils::enable_tracing;
+use std::sync::Arc;
+
+#[test]
+fn connect_with_client() {
```

Comment:
I'd like to do a TLS test too but I can't disable certificate checks when `ureq` is used directly inside the `get`/`put` functions

Reply:
I'm not worried. TLS is a really problem of our dependencies, not us.